Makefile

```
@@ -19,6 +19,42 @@ PACMAN_GLOB := $(CURDIR)/dist/$(PACMAN_PACKAGE_NAME)-[0-9]*.pkg.tar.*
 
 .DEFAULT_GOAL := help
 
+NATIVE_PKG_FORMAT_CMD = format=""; \
+os_release_token_match() { \
+	local expected token; \
+	for token in $${ID:-} $${ID_LIKE:-}; do \
```

checkmake (0.3.2)

[warning] Target body for "for token in $${ID" exceeds allowed length of 5 lines (8).

(maxbodylength)